pkg/cache/queue/manager.go

```diff
@@ -321,6 +323,7 @@ func (m *Manager) AddLocalQueue(ctx context.Context, q *kueue.LocalQueue) error
 		return fmt.Errorf("listing workloads that match the queue: %w", err)
 	}
 	for _, w := range workloads.Items {
+		m.assumedWorkloads[workload.Key(&w)] = key
```

Comment:
Hmm. At the first glance, being "assumed" has much in common with being "admitted":

https://github.com/kubernetes-sigs/kueue/blob/cc40ec1776dcb6fe98a8cd6741059ecfa1ea0496/pkg/scheduler/scheduler.go#L617-L625

Then, should _every_ workload pointing to this LQ be stored as "assumed"? Even e.g. inactive ones?

(Note: I'm not yet sure what I'm complaining about: maybe this line is right but just the field name is misleading?)